File: tests/indexing/parsing/lang/fixtures/java/Comments.java
public class Comments {

    /**
     * Documentation
     */
    public int withComments(int a, int b) {
        // a leading line comment
        int sum = a + b; // a trailing line comment
        /* a block comment
           spanning lines */
        String url = "http://not-a-comment";

        switch (code) {
            case 1:
                prepare();
                // a comment between statements
                dispatch();
        }

        return sum;
    }
}

File: tests/indexing/parsing/lang/fixtures/typescript/Comments.ts
/**
 * Combines two values.
 * @param a
 * @param b
 */
function withComments(a: number, b: number): number {
  // a leading line comment
  const sum = a + b; // a trailing line comment
  /* a block comment
     spanning lines */
  const url = "https://not-a-comment";
  return sum;
}
